app/views/gobierto_admin/gobierto_dashboards/dashboards/show.html.erb

```diff
@@ -1,3 +1,7 @@
 <div class="modal modal-full-width modal-no-padding">
-  <div dashboard-maker-app data-context="<%= @context %>" data-id="<%= @id %>"></div>
+  <div 
```

Comment:
If edit and show are the same, maybe they can be unified?

Reply:
Even with unified with edit and add an if to check if `data-id` should be rendered or not

Reply:
@Crashillo has made this change in #3629 removing all repeated views and using only one `modal` action in dashboards controller